src/SourcemapToolkit.SourcemapParser/StringExtensions.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace SourcemapToolkit.SourcemapParser
+{
+	internal static class StringExtensions
+	{
+		/// <summary>
+		/// String.Split() allocates an O(input.Length) int array, and
+		/// is surprisingly expensive.  For most cases this implementation
+		/// is faster and does fewer allocations.
+		/// There's a copy of this function in StandardFlightManager.cs because
+		/// that code can't reference this version - keep them in sync
+		/// </summary>
+		public static string[] SplitFast(this string input, char delimiter)
```

Comment:
Is there any way to avoid declaring this helper twice in the repo?

Reply:
In this repo?  It is only declared once.  It is shared with an implementation in a different repo, which we would need to dependency inject... but that would be kind of messy (IMVHO). Let me know what you think.

Reply:
src/SourcemapToolkit.SourcemapParser/StringExtensions.cs

and 
src/SourceMapToolkit.CallstackDeminifier/StringExtensions.cs